### PR TITLE
Force all resource groups within single collection via option

### PR DESCRIPTION
**Pull request by eabadjiev**
Hi there,

This is a backward compatible change that adds `--single-collection` option to the CLI. This new option will force all resource groups to be created as Folder within the same Postman collection.

FYI @bomiteva

### CODE REVIEW

**Comment by thecopy** [comment]
Hi, thanks for your contribution and sorry for this slow reply.

I think this is a great idea, but this breaks the app:

    $ apiary2postman.py --only-collection api some_api
    Traceback (most recent call last):
      File "./apiary2postman.py", line 123, in <module>
        main()
      File "./apiary2postman.py", line 120, in main
        write(input, output, args.only_collection, args.pretty, args.single_collection)
      File "/home/thecopy/dev/apiary2postman/apiary2postman/converter.py", line 83, in write
        result_out = _buildCollectionResponse(json_obj, single_collection)
      File "/home/thecopy/dev/apiary2postman/apiary2postman/converter.py", line 14, in _buildCollectionResponse
        single_collection)
      File "/home/thecopy/dev/apiary2postman/apiary2postman/converter.py", line 125, in parseResourceGroups
        for resourceGroup in apiary['resourceGroups']:
    TypeError: list indices must be integers, not str

**Comment by thim81** [comment]
Any news on this?
Will it get merged?

**Comment by eabadjiev** [comment]
Hey Erik, I have a very busy week, but happy to take a look next week if
that is OK.

Thanks,
E.

**Comment by thim81** [comment]
I just applied the PR to the latest version apiary2postman and it worked perfectly.
No error or warnings PLUS the output was one single collection which imported without a problem, keeping my postman a very clean workplace.

**Comment by thecopy** [comment]
@thim81 Yes, then added functionality works but it breaks already existing functionality.

@eabadjiev 
Change the first argument for the call to `parseResourceGroups` on line 10 to `apiary` from `apiary['resourceGroups']`

File: apiary2postman/converter.py [hunk 1]
```diff
@@ -3,14 +3,15 @@
 from uuid import uuid4
 from time import time
 
-def _buildCollectionResponse(apiary):
+def _buildCollectionResponse(apiary, single_collection):
 	environment = createEnvironment(apiary)
 
 	# Create the collection
 	collections = parseResourceGroups(
 		apiary['resourceGroups'],
```
Comment (thecopy) [inline apiary2postman/converter.py]: Change this to `apiary`

**Comment by thecopy** [comment]
@eabadjiev this is a 1 minute fix, if you could do it i will merge it asap

**Comment by eabadjiev** [comment]
@thecopy the change is applied. Also added few words to README.md.

**Pull request merged**